### DISCUSSION TITLE
How accurate is the estimated time to win?

### DISCUSSION THREAD

**Discussion by number435398** [Chia-Network/chia-blockchain#8037] [Q&A]
Recently, I've been winning about 4 times more regularly than I'm supposed to. This confuses me. I'm not complaining, I just want to distinguish between luck or imperfection in the estimate and or how to maintain what I'm getting.

**Comment by max4711qq** [comment]
This seems to be a regular problem unfortunately. Did you have regular wins before that were more or less in line with Estimated Time to Win? 4 times more than ETW should not be a problem if it happens here and there. 
I am experiencing much lower average wins than estimated. (Over one month now and my ETW is 2 days. So I am sure there is a "problem" out there. And I do win from time to time and my logs are OK. So it's not that I don't win at all. I am just not getting what I am promised to get)

**Reply by number435398** [reply]
A 2 day ETW and no wins in 30 days?! Have you checked your log files?  Maybe there's some error coming up.

**Reply by max4711qq** [reply]
I didn’t write those numbers.
Actually it’s one win in 25 days. And as I’ve stated several times: Logs are OK.
ETW 2 days

**Comment by BrandtH22** [comment]
@number435398 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note - the ETW is just an estimate and it is not uncommon to go 1/6th or even 6x the ETW.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
